pkg/rpc/stack_param.go

```diff
@@ -172,3 +172,32 @@ func (p *StackParam) UnmarshalJSON(data []byte) (err error) {
 	}
 	return
 }
+
+func (p *StackParam) TryParse(dest interface{}) error {
```

Comment:
`dest` can be [variadic](http://golang-club.blogspot.com/2016/09/go.html#variadic-functions)

that's would be very useful

Reply:
I think it would be better to add this functionality to `[]StackParam`

Reply:
`p *StackParam` rename all receivers to a single view

Reply:
Added func that parses variadic number of params.